Tom-Kelly-Answers.md

```diff
@@ -0,0 +1,14 @@
+Q1: So you're finishing up your time in this CSCC program. Why did you decide to embark on this journey?
+* *Tom:* I studied Computer Science at Ohio State, but I graduated nearly 8 years ago. I wanted to get a refresher on some of the new topics, as I have not been coding in quite a few years.
+
+Q2: I love to travel. Tell me a bit about your favorite travel destination so far. What's on your travel bucket list?
+* *Tom:* I traveled to Ireland last October, It was beautiful. I met some distant relatives in Ballycroy, County Mayo.
```

Comment:
I totally want to go to the British Isles.  Haven't been there, but it's on the list for sure!